app/routes/application-list.js

```diff
@@ -332,6 +361,18 @@ module.exports = router => {
         })
       }
 
+      if (subjects && subjects.length) {
+        selectedFilters.categories.push({
+          heading: { text: 'Subjects' },
```

Comment:
`Subject`

Reply:
Can you clarify whether you want the plural or singular form of 'subject'? I assume the singular, but other filter headings are plural like `Providers` and `Statuses`.